python/cudf/cudf/tests/test_spilling.py

```diff
@@ -594,3 +594,43 @@ def test_statistics_expose(manager: SpillManager):
     assert stat.count == 10
     assert stat.total_nbytes == buffers[0].nbytes * 10
     assert stat.spilled_nbytes == buffers[0].nbytes * 10
+
+
+def _statistics_rmm_fragmentation(size: int):
```

Comment:
Normally I like separate functions, but in this case the call is now decoupled from the original value of size and nothing prevents it from being called with a larger value. The logic here is tightly coupled to the logic of the test below. I would just inline it.